project.clj

```diff
@@ -69,19 +72,23 @@
   :cljsbuild {
     :builds [
       {:id "dev"
-       :source-paths ["src/cljs" "src/dev" "src/cljc"]
+       :source-paths ["src/cljs/nr" "src/cljs/dev" "src/cljc"]
        :figwheel true
-       :compiler {:output-to "resources/public/cljs/app.js"
+       :compiler {:output-to "resources/public/cljs/app10.js"
```

Comment:
Is this to force a cache invalidation for clients?

Reply:
That was the idea. There seemed to be some discussion on Nginx issues back when I started this.